src/content/general/architecture/capi/index.md

```diff
@@ -0,0 +1,227 @@
+---
+linkTitle: Cluster API architecture
+title: The Giant Swarm Platform Architecture
+description: Architecture overview explaining how our Platform is built and what services do we offer.
+weight: 30
+menu:
+  main:
+    parent: general-architecture
+last_review_date: 2022-09-12
+user_questions:
+  - Do you run a Developer Platform?
+aliases:
+  - /basics/platform-architecture/
+owner:
+  - https://github.com/orgs/giantswarm/teams/team-rocket
+---
+
+The Giant Swarm Platform consists of various systems. They can be categorized into three areas: infrastructure, applications, and operations.
```

Comment:
Please use sentence case everywhere, except for brand names. These are brand names:

- Giant Swarm
- Kubernetes
- Cluster API

These are not:

- cloud native
- platform